packages/fether-electron/src/main/app/menu/i18n/index.js

```diff
@@ -0,0 +1,77 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+import i18next from 'i18next';
+import Backend from 'i18next-node-fs-backend';
+import electron from 'electron';
+import settings from 'electron-settings';
+
+import Pino from '../../utils/pino';
+
+import { en } from './locales';
+
+let { app } = electron;
+const pino = Pino();
+
+const i18n = i18next;
+i18n
+  .use(Backend)
+  .init({
+    debug: true,
+    defaultNS: 'ns1',
```

Comment:
Yes that's correct.

I just had a look at https://www.i18next.com/principles/namespaces.

I think the translations in fether-electron should be renamed to `menu_ns`, and for the translations in fether-react to be renamed to `content_ns`. I suggest the underscore since that would be consistent with the keys in en.json, which are in snake case, and since in the code it would be used for example like `{i18n.t('content_ns:navigation.next_step')}`

I think it's ok to just use `ns` to stand for namespace, since the user can just do a quick search on the internet for the key `defaultNS` and the top search results will be the i18n docs such as 
https://www.i18next.com/overview/configuration-options, which defines it.

What do you think?

Reply:
Actually, perhaps we should use `fether_electron_menu_ns` and `fether_react_content_ns`

Reply:
Yes, agreed

Reply:
I've renamed it to the package name from package.json. And if a fether-react component uses a child component from fether-ui then it uses the fether-react namespace